pkg/plugins/backendplugin/grpcplugin/client_v2.go

```diff
@@ -191,6 +191,14 @@ func (c *ClientV2) QueryData(ctx context.Context, req *backend.QueryDataRequest)
 			return nil, plugins.ErrMethodNotImplemented
 		}
 
+		if status.Code(err) == codes.Unavailable {
+			return nil, plugins.ErrPluginGrpcConnectionTerminatedBase.Errorf("%v", err)
```

Comment:
Nit: I would be inclined to make this the same as the other (match the error code name)
```suggestion
			return nil, plugins.ErrPluginGrpcConnectionUnavailableBase.Errorf("%v", err)
```